components/brave_wallet/browser/tx_service.cc

```diff
@@ -269,7 +282,6 @@ void TxService::OnUnapprovedTxUpdated(mojom::TransactionInfoPtr tx_info) {
 }
 
 void TxService::Reset() {
-  ClearTxServiceProfilePrefs(prefs_);
   for (auto const& service : tx_manager_map_) {
     service.second->Reset();
   }
```

Comment:
I think we still should reset prefs if wallet gets reset and also should clear DB

Reply:
DB is actually being cleared [here](https://github.com/brave/brave-core/pull/11743/files#diff-ae1e5b4f7dfa5b814943fee796c7245bb6bad23365c86b4df4fd3db15e99e3acR149) but it will be called multiple times depends on coin type, I will move it to proper place.

Reply:
fixed in 330c37ab4a9a518d77a58769424ffe8b45e84776